src/main/scala/com/zeta/playthegame/repository/MongoConnection.scala

```diff
@@ -1,24 +1,31 @@
 package com.zeta.playthegame.repository
 
+import cats.effect.IO
 import com.typesafe.config.ConfigFactory
+import com.zeta.playthegame.repository.Entities.GameAppointmentDocument
 import org.mongodb.scala._
-import com.zeta.playthegame.repository.Entities._
-
-object MongoConnection {
 
+trait MongoConnection {
```

Comment:
El motivo por el cual existe este `Trait` es para poder extenderlo en los tests de integracion. Mi idea es que los tests de integracion usen el mismo mongo que se usaria en `dev` que seria el de docker. 
El primer problema que salta al hacer eso es que "esuciarias" tu mongo en modo `dev` con datos de los tests de integración, solución? escribir un `tearDown` que vuele los datos de los tests de integracion, pero nuevamente tendrias un problema que seria perder los datos de tu ambiente de `dev` porque harias un `dropCollection`.

Por eso mi idea es que en dev se use la collection definida en el objeto `MongoConnection` y que los IT creen una collection propiamente para su uso y luego poder hacer un `dropCollection` sin perder nada y no afectar a tus colecciones local. También se podria usar una mongo `db` distinta a la del ambiente `dev`

Reply:
Gracias por la aclaracion. Quizas sea bueno dejar un comment en el codigo acerca de esto.

Reply:
No se mucho como es mongo, pero no se solventa esto con un schema `test` 💃

Reply:
Yo tendria bases de datos distintas. Pero es una opinion apresurada.

Reply:
Me habia olvidado de que existia el `test` en mongo, de todas formas creo que sigue siendo necesario el trait este de MongoConnection, para luego definir un `TestMongoConnection` donde database sea `test` y las collections se mantengan o cambien kcyo.

Nose estoy pensando en voz alta

Reply:
`Nose estoy pensando en voz alta` jajaja ke malo.